library/src/main/java/com/getstream/sdk/chat/utils/MessageInputController.kt

```diff
@@ -200,11 +199,14 @@ class MessageInputController(
         }
     }
 
-    private suspend fun getAttachmentsFromLocal(isMedia: Boolean): List<AttachmentMetaData> =
+    private suspend fun getAttachmentsFromLocal(
+        isMedia: Boolean,
+        treeUri: Uri? = null
+    ): List<AttachmentMetaData> =
         withContext(Dispatchers.IO) {
             when (isMedia) {
-                true -> Utils.getMediaAttachments(view.context)
-                false -> Utils.getFileAttachments(Environment.getExternalStorageDirectory())
+                true -> StorageUtils.getMediaAttachments(view.context)
+                false -> StorageUtils.getFileAttachments(view.context, treeUri!!)
```

Comment:
We shouldn't use `!!` operator.
If we want that something won't be null, we should indicate it on the signature of the method.